frame/support/test/pallet/Cargo.toml

```diff
@@ -17,6 +17,7 @@ scale-info = { version = "2.0.0", default-features = false, features = ["derive"
 serde = { version = "1.0.163", default-features = false, features = ["derive"] }
 frame-support = { version = "4.0.0-dev", default-features = false, path = "../../" }
 frame-system = { version = "4.0.0-dev", default-features = false, path = "../../../system" }
+sp-runtime = { version = "24.0.0", default-features = false, path = "../../../../primitives/runtime" }
```

Comment:
Is it actually used?

Reply:
Yes, this is macro-generated code which uses `sp_runtime`:
```
1037     #[cfg(feature = "std")]
1038     impl<T: Config> frame_support::sp_runtime::BuildStorage for GenesisConfig<T> {
1039         fn assimilate_storage(
1040             &self,
1041             storage: &mut sp_runtime::Storage,
1042         ) -> std::result::Result<(), std::string::String> {
1043             frame_support::BasicExternalities::execute_with_storage(
1044                 storage,
1045                 || {
1046                     self.build();
1047                     Ok(())
1048                 },
1049             )
1050         }
1051     }
```